### PR TITLE
chore: improve `just final-check`

### PR DESCRIPTION
**Pull request by dpc**

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -777,6 +783,13 @@
                 + wasm32CrossEnvVars;
             });
 
+            # Like `cross` but only with wasm
```
Comment (justinmoon) [inline flake.nix:786]: `link`?

**Review by justinmoon — APPROVED** [review]

File: flake.nix (hunk 1 above)
Comment (dpc) [reply flake.nix:786]: The shell right above this one is called `cross`.

File: justfile [hunk 2]
```diff
@@ -30,7 +30,8 @@ lint:
   cargo doc --profile dev --no-deps --document-private-items
 
 final-check: lint
-  nix build -L .#cross.wasm32-unknown-unknown.client-pkgs
+  cargo test --doc
+  nix develop .#crossWasm -c cargo check --target wasm32-unknown-unknown --package mint-client
```
Comment (mxz42) [inline justfile:34]: do you want to add a comment above `final-check` when this is supposed to be used?

**Review by elsirion — APPROVED** [review]

File: justfile (hunk 2 above)
Comment (dpc) [reply justfile:34]: #1767

**Pull request merged**